plotly/tools.py

```diff
@@ -1670,6 +1722,321 @@ def create_streamline(x, y, u, v,
                              mode='lines', **kwargs)
         return streamline
 
+    @staticmethod
+    def create_ohlc(open, high, low, close, direction, **kwargs):
+
+        """
+        Returns data for an ohlc chart
```

Comment:
right now this actually returns a `trace` that "describes" an ohlc chart (is "describes" the right term to use here? thoughts @theengineear ?). multiple `traces` in a list is what we'd call `data` (but that term out of the figure context is pretty ambiguous)